res/raw/webxdc.js

```diff
@@ -39,5 +39,29 @@ window.webxdc = (() => {
     sendUpdate: (payload, descr) => {
       InternalJSApi.sendStatusUpdate(JSON.stringify(payload), descr);
     },
+
+    sendToChat: async (message) => {
+        const data = {};
+        if (!message.text && !message.file) {
+            return Promise.reject("Invalid empty message, at least one of text or file should be provided");
+        }
+        if (message.text) {
+            data.text = message.text;
+        }
+        if (message.file) {
+            let isString = value => typeof value === 'string' || value instanceof String;
```

Comment:
i read this as the definition of an anonymous function that is assigned to isString. if this this is correct, for better understanding, should't there be some brackets around the parameter list?

```suggestion
            let isString = (value) => typeof value === 'string' || value instanceof String;
```